packages/dart/sshnoports/bin/sshnp.dart

```diff
@@ -76,6 +92,20 @@ void main(List<String> args) async {
         ),
       );
 
+      storageDir = Directory(path.normalize('$homeDirectory'
+          '/${DefaultArgs.storagePathSubDirectory}'
+          '/${params.clientAtSign}'
+          '/storage'
+          '/${DateTime.now().millisecondsSinceEpoch}'));
+      storageDir!.createSync(recursive: true);
+
+      final sigintListener = ProcessSignal.sigint.watch().listen((signal) {
+        exitProgram(exitCode: 1);
+      });
+      ProcessSignal.sigterm.watch().listen((signal) {
+        exitProgram(exitCode: 1);
+      });
+
```

Comment:
Use a different storage directory every time. Listen for sigint and sigkill and call exitProgram if they are received